examples/docker/vault/deployment.yaml

```
@@ -0,0 +1,23 @@
+      labels:
+        app: docker-example
+    spec:
+      containers:
+      - name: app
+        image: nginx:latest
+        envFrom:
+        - secretRef:
+            name: docker-env-secrets
+        env:
+        - name: RUNTIME
+          value: "container" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)